cli/src/main.rs

```diff
@@ -25,25 +25,33 @@ fn main() -> Result<()> {
 
 	let config = std::fs::read_to_string(&config_path)?;
 
-	let (code, diagnostics) = run(config.as_str());
+	let ret = run(config.as_str());
+
+	let code = ret.code;
+	let diagnostics = ret.diagnostics;
 
 	if let Some(code) = code {
-		if let Some(definitions) = code.server.definitions {
-			let mut path = code.server.path.clone();
-			path.set_extension("d.ts");
+		let server_path = code.server.path.unwrap_or(PathBuf::from("net/server.lua"));
+		let client_path = code.client.path.unwrap_or(PathBuf::from("net/client.lua"));
```

Comment:
Is this a deliberate breaking change from `network` -> `net`?

Reply:
this isn't deliberate, I'll fix it